cypress/e2e/augury.cy.ts

```diff
@@ -57,4 +57,34 @@ describe("Augury", () => {
   it("creates flights and calculates inventory?", () => {
     // TODO
   });
+
+  it("creates and deletes creatives", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+    const audioFile = "cypress/samples/two-tone.mp3";
+
+    cy.visit("/creatives");
+    cy.get("h1").contains("Creatives");
```

Comment:
I believe this can be compacted to `cy.contains("h1", "Creatives");` and it will achieve the same thing